commands/search.js

```diff
@@ -72,7 +72,8 @@ class Search extends Command {
       const lureActive = effectIDs.includes(1);
 
       const searchText = lureActive ? 'walks through the tall grass with their Blob Lure and finds' : 'searches through the tall grass and finds';
-      const msg = await message.channel.send(`_${message.author} ${searchText}..._`);
+      let msg = await message.channel.send(`_${message.author} ${searchText}..._`);
+      message.delete().catch(x => {}); // eslint-disable-line no-unused-vars
```

Comment:
In theory, a better way to do this without eslint fuckery etc would be `.catch(() => {})`

Reply:
Also, is a catch really necessary here if we don't do it almost anywhere else in the file?

Reply:
Will change for the variable. We catch here (and not elsewhere) in case the user has already deleted their message or the bot doesn't have permission to. In contrast, the other messages are always done by the bot, so they should always still be there, and we should always have permission to delete them.